modules/ROOT/pages/custom-log-appender.adoc

```diff
@@ -7,32 +7,43 @@ endif::[]
 image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
 image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
-image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies", title="Runtime Fabric"]
+
+You can disable CloudHub logs and integrate your CloudHub application with your logging system using the Log4j configuration.
```

Comment:
```suggestion
You can disable CloudHub logs and integrate your CloudHub application with your logging system by using the Log4j configuration.
```
...just to clarify that "using the Log4j configuration" does not modify "system"....